app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,13 @@
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
+
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="ooni" />
```

Comment:
The intent filter for `run.ooni.io` is missing. See: https://developer.android.com/training/app-links/deep-linking.html

Reply:
It should look something like:

```
    <intent-filter android:label="@string/nettest">
        <data android:scheme="http"
              android:host="run.ooni.io"
              android:pathPrefix="/nettest" />
    </intent-filter>
```

Reply:
Also this intent filter for the `ooni` scheme is too broad. It should be restricted to `ooni://nettest` like so:

```
    <intent-filter android:label="@string/nettest">
        <data android:scheme="ooni"
              android:host="nettest" />
    </intent-filter>
```

Reply:
I didn't know we agreed on the http scheme other the ooni:// one, next time update specs so I don't lose what we write on the channel

Reply:
This is about adding support for universal links. It's the thing we discussed for hours and days :P

Reply:
The  android:host="nettest" is not needed as I check if the url is supported in the app, but I added it anyway.